docs/_static/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("theme.css");
+@import url("styles/furo.css");
```

Comment:
The css for indicators in darkmode is... shall I say... not great?
![Screenshot_20240429_111219](https://github.com/Ouranosinc/xclim/assets/91751417/be3b0552-68bc-4a23-bb1b-174d7c2a454b)